exercises/practice/palindrome-products/.articles/is-palindrome-check/content.md

```diff
@@ -0,0 +1,165 @@
+# Is number a palindrome? 
+
+There are multiple approaches to solve the palindrome product exercise. 
+Most of them require us to check if a given number is a palindrome. 
+
+## Approaches
```

Comment:
I'd remove the header (this is an article, and approaches might be confusing here).

```suggestion
```

Reply:
Once this is removed, the other headings can become level two headings.